### ISSUE TITLE
Add developer guide or links in readme

### ISSUE THREAD

**Issue by phyank** [neo-technology/front-end#3]
Is this project ready for production use? Is there any guide for developers?

**Comment by alastai** [comment]
This project is used in the Neo4j database product and in openCypher Morpheus (Cypher for Apache Spark) and Cypher for Gremlin projects. Please mail querylanguages@neo4j.com to follow up on your query? Thanks, Alastair

**Comment by bluejoe2008** [comment]
Can't agree any more.
I am also looking for docs telling how to implement a simple Cypher backend.

**Comment by crichey** [comment]
Yes I'm looking for any starting place on how to use the QueryParser. ANYTHING to point in the right direction would be very helpful.

**Comment by ghost** [comment]
Will be there guides in the future? I tried to add the parser into my project and there was lots of problems. Most of them was static references to non-static types. I fixed them fast and dirty to test my project and have find out an fully implemented `ASTFactory` is missing. There's only the interface of it. The `java` files was generated with JavaCC and `cypher.jj`. Thanks in advance.
